src/nunit.analyzers.tests/UpdateStringFormatToInterpolatableString/UpdateStringFormatToInterpolatableStringCodeFixTests.cs

```diff
@@ -47,6 +47,26 @@ public void AccidentallyUseFormatSpecification(string assertOrAssume)
         }}");
             RoslynAssert.CodeFix(analyzer, fix, expectedDiagnostic, code, fixedCode);
         }
+
+        [TestCaseSource(nameof(AssertAndAssume))]
+        public void AccidentallyUseFormatSpecificationWithVariable(string assertOrAssume)
```

Comment:
The `Accidentally` here is because this code does compile on NUnit 4, but calls a different signature than expected:

![image](https://github.com/nunit/nunit.analyzers/assets/8655218/ed2b289d-b5ec-42cf-878c-681b32160484)

When failing it shows:
![image](https://github.com/nunit/nunit.analyzers/assets/8655218/82183df2-c737-4207-89e8-4ede3081c738)

instead of:
![image](https://github.com/nunit/nunit.analyzers/assets/8655218/d0e6b443-ddc5-4fcd-b238-0c1357cb7aaa)